src/plugins/workspace/public/components/data_source_association/association_data_source_modal.tsx

```diff
@@ -25,7 +25,11 @@ import {
 import { FormattedMessage } from 'react-intl';
 import { i18n } from '@osd/i18n';
 
-import { getDataSourcesList, fetchDataSourceConnections } from '../../utils';
```

Comment:
I think we can remove function `fetchDataSourceConnections` if it's not used anymore.

Reply:
fetchDataSourceConnections is also required in DataSourceTable, so perhaps we should keep it.